src/NServiceBus.AcceptanceTests/EndpointTemplates/DefaultServer.cs

```diff
@@ -61,6 +61,23 @@ public DefaultServer(List<Type> typesToInclude)
             await builder.DefinePersistence(settings);
 
             builder.GetSettings().SetDefault("ScaleOut.UseSingleBrokerQueue", true);
+            builder.NotifyOnFailedMessage(failedMessage =>
```

Comment:
> so would making it an "add new action" api meet your future requirements?

yes I think that should be fine. I can't come up with subscrubing/unsubscribing dynamically at runtime (although at feature setup time).

> I want to avoid making it an event since that implies it can be added or removed at runtime

out of curiosity: what's the problem by adding/removing at runtime besides the null check (and race conditions when subscribing/unsubscribing)?